src/main/webapp/jsp/instructorSearch.jsp

```diff
@@ -278,14 +278,17 @@
                                                         responseCommentIndex++;
                                                         String frCommentGiver = data.feedbackResponseCommentSearchResultBundle
                                                                                          .commentGiverTable.get(frc.getId().toString());
+                                                        if (!frCommentGiver.equals("Anonymous")) {
+                                                            frCommentGiver = frc.giverEmail;
+                                                        }
                                                 %>
                                                 <li class="list-group-item list-group-item-warning"
                                                     id="responseCommentRow-<%=fsIndx%>-<%=qnIndx%>-<%=responseIndex%>-<%=responseCommentIndex%>">
                                                     <div id="commentBar-<%=fsIndx%>-<%=qnIndx%>-<%=responseIndex%>-<%=responseCommentIndex%>">
                                                         <span class="text-muted">
                                                             From:
-                                                            <b><%=frCommentGiver%></b>
-                                                            on <%=TimeHelper.formatTime(frc.createdAt)%>
+                                                            <%= frCommentGiver %>
+                                                            [<%= frc.createdAt %>] <%=frc.getEditedAtText(frc.giverEmail.equals("Anonymous"))%>
```

Comment:
If this is the reason you opened #3867, I advice you to dig deeper into the back-end logic. There is already a part that changes the `frc.giverEmail` to `"Anonymous"` after retrieving from database.